### ISSUE TITLE
Window (Scrollable)

### ISSUE THREAD

**Issue by ModeAxe** [adicu/DevFest-2023#5]

**Comment by ethmth** [comment]
Hi this might be a silly question but is there any more specific description of this functionality? To me it seems like this means that within each desktop window, you can scroll if the content exceeds the window height. It seems like this functionality is already implemented (I don't think I did anything special to make it this way, it just seems like it by default). If this is not what it means or I'm supposed to do something else, I'd be happy to! Just please let me know.

**Comment by ModeAxe** [comment]
Yeah, the description here is a bit unclear. Default window is supposed change it’s height to fit content and this one is supposed to have a fixed height and let users scroll. I believe your test window is already an example of this. One of the reasons this is a separate work item is because the styling of the scroll bar needs to change to match the design document.
